recipes/faiss/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "1.6.3" %}
+
+package:
+  name: faiss
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/faiss/archive/v{{ version }}.tar.gz
+  sha256: e1a41c159f0b896975fbb133e0240a233af5c9286c09a28fde6aefff5336e542
+
+build:
+  number: 0
+  skip: True  # [win]
+
+# Need these up here for conda-smithy to handle them properly.
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+  run:
+    - python
+
+outputs:
+  - name: faiss-cpu
+    script: build-cpu.sh
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - swig
+        - cmake
+      host:
+        - python
+        - pip
+        - blas =*=mkl
+        - mkl >=2018
+        - numpy =1.14.*
+      run:
+        - blas =*=mkl
+        - mkl >=2018
+        - numpy
+        - llvm-openmp  # [osx]
+        - libgomp      # [linux]
```

Comment:
These should not be in `run`. Please see the docs.

Reply:
Ok, adding them to build-reqs. Docs [say](https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp): 
> You can enable OpenMP on macOS by adding the llvm-openmp package to the build, host, and run sections of the meta.yaml.

But I'm guessing this has strong run-exports?

Reply:
the comment anchor here is a bit confused (underlying file has changed a lot); in the case of `faiss-cpu`, this failed to resolve with `llvm-openmp` and `libgomp` in the build requirements, but works in host-reqs.